cdap-proto/src/main/java/co/cask/cdap/proto/profile/Profile.java

```diff
@@ -32,7 +32,7 @@
 public class Profile {
   public static final Profile NATIVE = new Profile("native", "native", "Runs programs locally on the cluster",
                                                    EntityScope.SYSTEM,
-                                                   new ProvisionerInfo("yarn", Collections.emptyList()));
+                                                   new ProvisionerInfo("native", Collections.emptyList()));
```

Comment:
can you also extract 'native' out into a variable and use that everywhere the string is used?